src/mngt/ocf_mngt_cache.c

```diff
@@ -1654,14 +1659,8 @@ int ocf_mngt_cache_start(ocf_ctx_t ctx, ocf_cache_t *cache,
 		ocf_cache_log(*cache, log_info, "Successfully added\n");
 		ocf_cache_log(*cache, log_info, "Cache mode : %s\n",
 			_ocf_cache_mode_get_name(ocf_cache_get_mode(*cache)));
-	} else {
-		if (cfg->name) {
-			ocf_log(ctx, log_err, "Inserting cache %s failed\n",
-					cfg->name);
-		} else {
-			ocf_log(ctx, log_err, "Inserting cache failed\n");
-		}
-	}
+	} else
+		ocf_log(ctx, log_err, "Inserting cache failed\n");
```

Comment:
As cache name is mandatory now, I'd rather leave the variant with name, which is more informative.